Mage.Client/src/main/java/mage/client/MageFrame.form

```diff
@@ -84,7 +84,6 @@
     </Container>
     <Container class="javax.swing.JToolBar" name="mageToolbar">
       <Properties>
-        <Property name="floatable" type="boolean" value="false"/>
```

Comment:
?

Reply:
Odd, this change must have happened during a merge. I don't even see the .form files in my IDE (Netbean), so I'm not sure how to roll it back.

Reply:
Fixed it on github, all you need to do now is pull